assignments/colorpicker.md

```diff
@@ -132,60 +126,70 @@ class I normal
 
 </div>
 
-We'll handle touch input by implementing `onTouchEvent`. This is the event handler that will be called when a touch occurs in this view. In addition, when feedback is needed, you will have to ensure that the view is _redrawn_.
+Note that the End state only exists to show the lifetime of a _single_ interaction. Because the user can interact with the color picker any number of times, we would actually return to the Start state when the thumb is released. For some examples of single interactions, see the diagrams below.
+
+We'll handle touch input by implementing `onTouchEvent`. This is the event handler that will be called when a touch occurs in this view. In addition, when feedback is needed, you will have to ensure that the view is _redrawn_. For style purposes, you should only redraw the view when necessary. We **will be taking off points** for redraw calls if they are not necessary. A good general guideline is to only redraw in your PPS code.
```

Comment:
should we say that you should also make sure that invalidate is only called at most once per cycle (per call of onTouchEvent), likely has no impact - and doesn't even trigger additional redraws - but good style?